TelepresenceBot/server-unit/core/loggingClient.js

```diff
@@ -0,0 +1,14 @@
+function LoggingClient(client) {
+    this.client = client;
+}
+
+LoggingClient.prototype.emit = function(event, dataToEmit) {
```

Comment:
This function mirrors the socket io `client` so that for real runs of the server we use this logging client and for testing we can use an actual socket io `client`.